CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/dragdrop/FileDropper.java

```diff
@@ -105,4 +82,42 @@ public void execute(final PluginGraphs graphs, final PluginInteraction interacti
 
         return null;
     }
+
+    @PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
+    public static class DragAndDropFilePlugin extends SimplePlugin {
```

Comment:
given this plugin is opening a file via drag and drop, I'm not sure MODIFY is the tag to use. Maybe UTILITY (or GENERAL if nothing else fits)? Plugin type might need to be different as well

Reply:
not import ?

Reply:
The parent plugin is an IMPORT. Should these also be Import Type and IMPORT
tag?  
PluginType.CREATE, tags = {"IMPORT"} Or 
PluginType.IMPORT, tags = {"IMPORT"}

Reply:
OR PluginType.IMPORT, tags = {"UTILITY "}?

Reply:
Personally I'd say PluginType.IMPORT, tags = {"IMPORT"} fits best

Reply:
@Auriga2 I'd go with however we have tagged opening graphs elsewhere (bar WELCOME since that wouldn't make sense in this instance). I did think about IMPORT but thought that this really opening a graph rather than importing one. If IMPORT though is how we've tagged it elsewhere then go with that

Reply:
I am not sure how to find these. I can see some with PluginType.DISPLAY, tags = {"MODIFY"}.
Shall I add multiples in that case? Say PluginType.CREATE, tags = {"IMPORT", "UTILITY "}?

Reply:
`OpenFilePlugin` is tagged as `@PluginInfo(pluginType = PluginType.IMPORT, tags = {"LOW LEVEL"})`. I'd say that is probably the most similar action so I would go with that

Reply:
Sorry but I would go with `IMPORT` because they are part of the same plugin. It would also mean that when you filter in the notes view for `IMPORT` then there will still be a lingering plugin for `MODIFY` which was for the same action. That is, effectively you have both `IMPORT` and `MODIFY` as tags.

Reply:
Not Clear, do you mean ` PluginType.IMPORT, tags = {"IMPORT"}`?

Reply:
@arcturus2 if that is the case, should we change the `OpenFilePlugin` tags then? These are more or less doing the same thing (one opens graph via file explorer, and the other via drag and drop), so having different tags for the pair them doesn't make sense

Reply:
Yep @antares1470, opening a file would be `IMPORT` to my mind.

> Not Clear, do you mean PluginType.IMPORT, tags = {"IMPORT"}?

Yes @Auriga2